ops/internal/manage/chains.md.tmpl

```diff
@@ -0,0 +1,11 @@
+# Superchain Ecosystem
+
+{{ range $index, $superchain := .Superchains -}}
+### {{ $superchain }}
+
+| Chain Name | OP Governed | Upgradeable | Explorer | Public RPC | Sequencer RPC
+|---|---|---|---|---|---|
+{{- range index $.ChainData $index }}
+| {{ .Name }} | {{ checkmark .GovernedByOptimism }} | {{ optedInSuperchain .SuperchainTime }} | {{ .Explorer }} | `{{ .PublicRPC }}` | `{{ .SequencerRPC }}` |
```

Comment:
How does the `chainConfig.GovernedByOptimism` value get set now? Will that be part of the chain's intent.toml or state.json file? I'm wondering if we should remove that boolean from the ChainConfig struct and dynamically determine if the chain is currently governed by optimism when generating the CHAINS.md file instead of relying on a static field

Reply:
It's set in `meta.toml`.